lib/player.js

```diff
@@ -724,6 +728,9 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       };
     });
 
+    // Stop the variant checker timer
+    this.checkVariantsTimer_.stop();
```

Comment:
This isn't necessary. As part of the destroy process (see the `await` below) we are detaching the player, and the first set of detaching is to unload. So the `stop` call in `onUnload_` should already happen as part of this method.